blogs/whats_new_1.5/index.md

```diff
@@ -0,0 +1,178 @@
+
+### Overview tab
+
+Graphs for Flow RTT and DNS Tracking, including support for DNS TCP (previously only DNS UDP), were added.  There are graphs for:
+
+- Top 5
+- Bottom 5
+- 90th percentile or the top 10% (P90)
+- 99th percentile or the top 1% (P99)
+
+#### Manage panels selection
+
+With so many graphs to choose from, the **Manage panels** dialog, found under "Show advanced options", now provides a simple filter (Figure 5).  Click one or more buttons to filter on the selection.
+
+![Manage Panels](images/manage_panels.png)
+_<div style="text-align: center">Figure 5: Manage panels</div>_
+
+#### Graph types
+
+If you click **⋮** in the upper right corner of the graph, there will be various options depending on the type of graph.  For example, the ***TCP handshake Round Trip Time*** graph shows a donut chart but can be changed to use a lines graph (Figure 6).
+
+<img src="images/graph_type-donut.png" alt="donut graph" width="450"> <img src="images/graph_type-lines.png" alt="lines graph" width="450">
+_<div style="text-align: center">Figure 6: Options - Graph type</div>_
+
+#### Single graph focus
+
+In Overview, it displays all graphs on a scrollable panel.  If you click the "Focus" icon in the upper right corner next to **⋮**, it displays one graph and gives a preview of all the other graphs on a scrollable panel on the right side (Figure 7).  If you click a preview graph, it becomes the graph in focus.  This feature can also be toggled in the "Display options" dropdown.
+
+![single graph focus](images/single_graph_focus.png)
+_<div style="text-align: center">Figure 7: Single graph focus</div>_
+
+### Traffic flows tab
+
+These are the new labels in the flow data.
+
+1. Differentiated Services Code Point (DSCP)<br>
+This is a 6-bit value in the IP packet header that indicates the priority of a packet to provide quality of service (QoS), particularly for time-sensitive data such as voice and video.  The value "Standard" translates to 0 or best effort.  In other words, the traffic is not getting any special treatment.
+
+    - Column: DSCP
+    - Label: Dscp
+
+![flow table - DSCP](images/flow_table-dscp.png)
+_<div style="text-align: center">Figure 8: DSCP</div>_
+
+2. Availability Zones<br>
+A region defines a geographical area and consists of one or more availability zones.  For example, if a region is named us-west-1, then the zones might be us-west-1a, us-west-1b, and us-west-1c, where each zone might have one or more physical data centers.
+
+    - Columns: Source Zone, Destination Zone
+    - Labels: SrcK8S_Zone, DstK8S_Zone
+
+![flow table - Zone](images/flow_table-zone.png)
+_<div style="text-align: center">Figure 9: Availability Zone</div>_
+
+3. Cluster ID<br>
+The cluster ID is the same value shown in the **Home > Overview**, Details section.
+
+    - Column: Cluster
+    - Label: K8S_ClusterName
+
+![flow table - Cluster ID](images/flow_table-cluster_id.png)
+_<div style="text-align: center">Figure 10: Cluster ID</div>_
+
```

Comment:
You can add a mention to the full list of fields here:
https://github.com/netobserv/network-observability-operator/blob/main/docs/flows-format.adoc